server/player/player.go

```diff
@@ -1160,6 +1205,75 @@ func (p *Player) Jump() {
 	}
 }
 
+// Sleep makes the player sleep at the given position. If the position does not map to a bed (specifically the head side),
+// the player will not sleep.
```

Comment:
We try not to go over 110 characters in a single line of docs
```suggestion
// Sleep makes the player sleep at the given position. If the position does not map to a bed (specifically
the head side), the player will not sleep.
```